src/Core/Logger.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * @file src/Core/Logger.php
+ */
+namespace Friendica\Core;
+
+use Friendica\Core\Config;
+use Friendica\Util\DateTimeFormat;
+
+class Logger
+{
+    /**
+     * @brief Logs the given message at the given log level
+     *
+     * log levels:
+     * LOGGER_WARNING
+     * LOGGER_INFO (default)
+     * LOGGER_TRACE
+     * LOGGER_DEBUG
+     * LOGGER_DATA
+     * LOGGER_ALL
+     *
+     * @global array $LOGGER_LEVELS
+     * @param string $msg
+     * @param int $level
+     */
+    public static function log($msg, $level = LOGGER_INFO)
+    {
+        $a = get_app();
```

Comment:
`get_app()` shouldn't be used.

You can use extend Logger from BaseObject like
```php
class Logger extends BaseObject
```
and than use
```php
$a = self::getApp();
```